mssql/README.md

```diff
@@ -0,0 +1,38 @@
+# Spice Demo with SalesLT dataset in MSSQL
+
+This demo showcases how to use Spice with the SalesLT sample dataset in MSSQL.
+
+## Prerequisites
+
+- MSSQL Server with SalesLT sample dataset
```

Comment:
No one knows what this is, link and description on how to set up.

Reply:
Done